tests/test_admin.py

```diff
@@ -63,20 +63,19 @@ def test_language_filter(self):
 class AliasContentManagerTestCase(CMSTestCase):
     def setUp(self):
         self.superuser = self.get_superuser()
-
-    @skipUnless(not is_versioning_enabled(), 'Test only relevant when no versioning')
-    def test_alias_content_manager_rendering_without_versioning_actions(self):
-        category = Category.objects.create(name='Language Filter Category')
```

Comment:
It's fine to repeat these things sometimes because when you add it to the setUp method you effectively create the record for every unit test, regardless of whether you use it or not. This can weigh on test suites over time when more and more unit tests are added to a TestCase class that then don't use the values in the setup. 

This is fine to leave now but worth considering in the future.